src/jdk.jlink/share/classes/jdk/tools/jimage/JImageTask.java

```diff
@@ -211,7 +211,7 @@ int run(String[] args) {
                 } else {
                     try {
                         log.println(TASK_HELPER.getMessage("main.usage." +
-                                options.task.toString().toLowerCase()));
+                                options.task.toString().toLowerCase(Locale.ROOT)));
```

Comment:
This is the lookup of the main.usage.list resource (`jimage list --help`) rather than the `list` option but good to fix this anyway.

Reply:
> This is the lookup of the main.usage.list resource (`jimage list --help`) rather than the `list` option but good to fix this anyway.

Sorry, I found the problem with text search, but didn't look at them carefully.